src/material.rs

```diff
@@ -54,6 +62,8 @@ pub mod basic {
 }
 
 /// Parameters for a Lamberian diffusion reflection model.
+///
+/// Renders triangle meshes with the Gouraud illumination model.
```

Comment:
*Lambertian illumination model.

Gouraud _shading_ refers to whether lighting computed at the _vertices_ of a polygon are interpolated across the face, hence the `flat` field disables/enables Gouraud shading.

Reply:
I considered having an `enum Shading { Flat, Gouraud }` but that felt a bit overkill.

Reply:
Sure, I've been moving doc lines forth and back, so it's not the last change.